traitlets/config/application.py

```diff
@@ -151,7 +151,7 @@ def _classes_inc_parents(self):
     argv = List()
 
     # Whether failing to load config files should prevent startup
-    raise_config_file_errors = Bool(False)
+    raise_config_file_errors = Bool(os.environ.get('TRAITLETS_APPLICATION_RAISE_CONFIG_FILE_ERROR', '').lower() == 'true')
```

Comment:
The others ENV variables are module wise and `_load_config_files` is a class method, so it is use without instance (like in IPython/application.py) so I'm unsure.